server/src/modules/collections/collections.service.ts

```diff
@@ -1047,7 +1047,7 @@ export class CollectionsService {
     try {
       const result = await this.plexApi.getCollection(id);
 
-      if (result.smart) {
+      if (result?.smart) {
```

Comment:
This fixes an exception being thrown when the collection does not exist. The same value is returned so there is no functional difference, just stops an error log message.